src/python/pants/backend/python/rules/inject_init_test.py

```diff
@@ -23,11 +25,18 @@ def rules(cls):
             RootRule(Digest),
         )
 
-    def assert_injected(self, *, original_files: List[str], expected_added: List[str]) -> None:
+    def assert_injected(self, *, original_files: List[str], expected_added: List[str],
+                        sources_stripped=True) -> None:
         request = InjectInitRequest(
-            self.make_snapshot({fp: "# python code" for fp in original_files})
+            self.make_snapshot({fp: "# python code" for fp in original_files}),
+            sources_stripped=sources_stripped,
         )
-        result = self.request_single_product(InitInjectedSnapshot, request).snapshot
+        result = self.request_single_product(
+            InitInjectedSnapshot,
+            Params(
+                request, create_options_bootstrapper(args=["--source-root-patterns=['src/python']"])
```

Comment:
I think you can get rid of the argument, now that we have default source roots.

Reply:
I'd rather not rely on defaults in a test.